src/rcheevos/trigger.c

```diff
@@ -18,6 +18,8 @@ void rc_parse_trigger_internal(rc_trigger_t* self, int* ret, void* buffer, rc_sc
     if (*ret < 0) {
       return;
     }
+
+    self->requirement->next = 0;
```

Comment:
This just ensures the core group's next pointer is initialized. As far as I could tell, nothing was trying to iterate the core group.